### PR TITLE
Add docs about hardware design

### LINKED ISSUE #91: Create documentation about AMDC hardware design
- Power Distribution: Explain all this related to AMDC power... voltage rails, connections, limitations, etc.
- Power Stack I/O: What are the connector I/O, power rails available, jumper selection, etc...
- Analog Inputs:
- isoSPI:
- Encoder:

### PR DESCRIPTION
**Pull request by npetersen2**
Resolves #91

### CODE REVIEW

File: docs/PowerDistribution.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+The design requirements for the power input and distribution on the AMDC can be boiled down to a succient list of conflicting goals. The remainder of this document will summarize the resulting AMDC power system design and how it adheres to these goals. The following describes these goals:
+
+1. Single voltage input connection for all PCB power.
+2. Efficient DC/DC conversion for various voltage rails (steady state should not be excessively hot for devices).
+3. Low noise on power rails for circuitry (switching DC/DC are noisy; LDOs are preferred).
+4. Power subsystem is not the main goal of AMDC, therefore BOM should be small and inexpensive.
+
+## Block Diagram
+
+<img src="images/amdc-power-distribution.svg" />
```
Comment (elsevers) [inline docs/PowerDistribution.md:16]: Questions about this figure:

- What are the `OR` blocks? Are these each LDO's? If so, perhaps the pink blocks should reflect the trimmed output voltage of the DC/DC regulator? 

- I am confused about what the pink blocks represent, they seem to have the same name as the text on the right side of the image. 

- There are several gray blocks that look like switches. What are these? Jumpers?

**Review by elsevers — CHANGES_REQUESTED** [review]
This looks very thorough. Nice job.

Please add a description of power routing to the inverter outputs and external measurement circuitry (ADC ports) that comes from the 24V input. I understand that the details of this belong in different docs (i.e. a markdown file on inverter output), but we should acknowledge that in here and provide a link to those docs.

As for as a template, let's consider how we want to denote differences in REVs of the AMDC. Do we want to have a dedicated differences section of each markdown document? Or call it out as we go along in the description? Let's keep the document oriented toward the current revision of the AMDC (noting that people can look at a previous commit to see the document oriented toward a previous revision), but we should still call out major differences. 


When we revise one of these documents for a new board rev, we can provide a link to a "finalized" version of this document for a previous board rev.

File: docs/PowerDistribution.md (hunk 1 above)
Comment (npetersen2) [reply docs/PowerDistribution.md:16]: I have updated the block diagram, hopefully it is more clear now.

**Comment by npetersen2** [comment]
@elsevers I have updated the doc to include your suggestions. Please review.

For different PCB revisions, I think just calling out differences in the doc as they come up will work well. Perhaps each doc should have a "change log" at the bottom for revision to revision changes as well, so users don't have to skim the entire doc for things (kind of like last page of the REV D schematics).

**Review by elsevers — APPROVED** [review]
Looks good.

**Pull request merged**